package.json

```diff
@@ -87,6 +87,7 @@
     "react-typing-animation": "^1.6.2",
     "react-youtube-embed": "^1.0.3",
     "reading-time": "^1.2.0",
+    "recharts": "^2.7.2",
```

Comment:
Have you added this library using proper npm versions as mentioned in `README.md`?

Reply:
@vishvamsinh28

Reply:
@vishvamsinh28 Kindly address this comment.

Reply:
@vishvamsinh28 comment is still not resolved!!!